app/views/person_mailer/welcome_email.html.erb

```diff
@@ -2,6 +2,9 @@
 <p/>
 This message contains important information regarding your CMU Silicon Valley email account.
 <p/>
+It is a copy (perhaps outdated) of the information at
```

Comment:
Wanted to include that information in the email message because people tend not to follow a link to read, even if explicitly told to do so.  (Had that happen for 2 out 5 people this week, in fact.)  Will reword that a little, though.